developer/help/how-to-add-the-cmdlet-name-and-synopsis-to-a-cmdlet-help-topic.md

```diff
@@ -10,75 +10,26 @@ ms.assetid: 1d0e1eb1-a962-4406-9625-175cfa3364ad
-      <maml:description>
-    </command:details>
-    ```
-
- Here are some things to remember when adding the synopsis of a cmdlet.
-
--   The cmdlet Help can display two descriptions for the cmdlet. The first description is a short description that is referred to as the synopsis. The second description is a more detailed description that is discussed in [Adding the Detailed Description to a Cmdlet Help Topic](./how-to-add-a-cmdlet-description.md). Both these descriptions should be written as a single paragraph.
+- The cmdlet Help can display two descriptions for the cmdlet. The first description is a short description that is referred to as the synopsis. The second description is a more detailed description that is discussed in [Adding the Detailed Description to a Cmdlet Help Topic](./how-to-add-a-cmdlet-description.md). Both these descriptions should be written as a single paragraph.
 
--   In the synopsis do not repeat the cmdlet name. Informing the user that the Get-Server cmdlet gets a server is brief, but not informative. Instead, use synonyms and add details to the description.
+- In the synopsis do not repeat the cmdlet name. Informing the user that the Get-Server cmdlet gets a server is brief, but not informative. Instead, use synonyms and add details to the description.
 
-     Example: "Gets an object that represents a local or remote computer."
+  Example: "Gets an object that represents a local or remote computer."
 
--   Use simple verbs like "get", "create", and "change" in the synopsis. Avoid using "set" because it is vague, and fancy words such as "modify."
+- Use simple verbs like "get", "create", and "change" in the synopsis. Avoid using "set" because it is vague, and fancy words such as "modify."
 
-     Example: "Gets information about the Authenticode signature in a file."
+  Example: "Gets information about the Authenticode signature in a file."
 
--   Write in active voice. For example, "Use the TimeSpan object..." is much clearer than "the TimeSpan object can be used to..."
+- Write in active voice. For example, "Use the TimeSpan object..." is much clearer than "the TimeSpan object can be used to..."
 
--   Avoid the verb "display" when describing cmdlets that get objects. Although Windows PowerShell displays cmdlet data, it is important to introduce users to the concept that the cmdlet returns .NET Framework objects whose data may not be displayed. If you emphasize the display, the user might not realize that the cmdlet may have returned many other useful properties and methods that are not displayed.
+- Avoid the verb "display" when describing cmdlets that get objects. Although Windows PowerShell displays cmdlet data, it is important to introduce users to the concept that the cmdlet returns .NET Framework objects whose data may not be displayed. If you emphasize the display, the user might not realize that the cmdlet may have returned many other useful properties and methods that are not displayed.
 
 ## See Also
```

Comment:
There are no links here. What happened?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.